docs/rum_collection/data_collected.md

```diff
@@ -9,13 +9,26 @@ further_reading:
     tag: "Documentation"
     text: "Datadog Real User Monitoring"
 ---
-The RUM SDK generates events that have associated metrics and attributes. Metrics are quantifiable values that can be used for measurements related to the event. Attributes are non-quantifiable values used to slice metrics data (group by) in analytics.
+
+## Overview
+
+The RUM iOS SDK generates events that have associated metrics and attributes. Metrics are quantifiable values that can be used for measurements related to the event. Attributes are non-quantifiable values used to slice metrics data (group by) in analytics. 
 
 Every RUM event has all of the [default attributes](#default-attributes), for example, the device type (`device.type`) and user information such as their name (`usr.name`) and their country (`geo.country`).
 
 There are additional [metrics and attributes that are specific to a given event type](#event-specific-metrics-and-attributes). For example, the metric `view.time_spent` is associated with "view" events and the attribute `resource.method` is associated with "resource" events.
 
-This page provides descriptions of each of the metrics and attributes collected.
+| Event Type | Retention | Description                         |
+|------------|-----------|-------------------------------------|
+| Session    | 30 days   | A session represents a real user journey on your mobile application. It begins when the user launches the application, and the session remains live as long as the user stays active. During the user journey, all RUM events generated as part of the session will share the same `session.id` attribute. |
+| View       | 30 days   | A view represents a unique screen (or screen segment) on your mobile application. Individual `UIViewControllers` are classified as distinct views. While a user stays on a view, RUM event attributes (Errors, Resources, Actions) get attached to the view with a unique `view.id`.                           |
```

Comment:
It might be interesting to include the exact lifecycle events we use to define when a view starts and ends here. (cc @ncreated @maxep )

Reply:
Good idea 👍. The way it works is that View start and stop are notified by `viewDidAppear(animated:)` and `viewDidDisappear(animated:)` callbacks (on `UIViewController` class).

Reply:
```suggestion
| View       | 30 days   | A view represents a unique screen (or screen segment) on your mobile application. A view starts and stops when the `viewDidAppear(animated:)` and `viewDidDisappear(animated:)` callbacks on the UIViewController class are notified. Individual `UIViewControllers` are classified as distinct views. While a user stays on a view, RUM event attributes (Errors, Resources, Actions) get attached to the view with a unique `view.id`.                           |
```